templates/sig/prism/node.rbs.erb

```diff
@@ -33,6 +33,7 @@ module Prism
     def fields: () -> Array[Prism::Reflection::Field]
     def inspect: () -> String
     def type: () -> Symbol
+    def deprecated: (*String) -> void
```

Comment:
Can you put this on `class Node`? `_Node` is for the stuff that is implemented on each node.

Reply:
OK, I'll try it!